### ISSUE TITLE
UndefVarError: unsafe_load! not defined

### ISSUE THREAD

**Issue by fleimgruber** [analytech-solutions/CBinding.jl#86]
On https://github.com/fleimgruber/DWDataReader.jl when running `julia test/runtests.jl` I get the traceback below. This is on GNU/Linux (NixOS) using Julia 1.5.1. Tests were developed on Windows OS and do not trigger the error there.

```
WDataReader.File: Error During Test at /home/fps/dev/DWDataReader.jl/test/runtests.jl:11
  Got exception outside of a @test
  LoadError: UndefVarError: unsafe_load! not defined
  Stacktrace:
   [1] getindex(::CBinding.Cptr{var"(c\"struct DWChannel\")"}, ::Int64) at /home/fps/.julia/packages/CBinding/BsBOA/src/pointers.jl:54
   [2] (::DWDataReader.var"#2#3"{CBinding.Cptr{var"(c\"struct DWChannel\")"}})(::Int64) at ./none:0
   [3] iterate at ./generator.jl:47 [inlined]
   [4] _all at ./reduce.jl:827 [inlined]
   [5] all at ./reduce.jl:823 [inlined]
   [6] Dict(::Base.Generator{UnitRange{Int64},DWDataReader.var"#2#3"{CBinding.Cptr{var"(c\"struct DWChannel\")"}}}) at ./dict.jl:130
   [7] DWDataReader.File(::String; debug::Bool, kw::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/fps/dev/DWDataReader.jl/src/file.jl:119
   [8] DWDataReader.File(::String) at /home/fps/dev/DWDataReader.jl/src/file.jl:81
   [9] top-level scope at /home/fps/dev/DWDataReader.jl/test/testfiles.jl:6
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at /home/fps/dev/DWDataReader.jl/test/runtests.jl:14
   [12] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1115
   [13] top-level scope at /home/fps/dev/DWDataReader.jl/test/runtests.jl:13
   [14] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1115
   [15] top-level scope at /home/fps/dev/DWDataReader.jl/test/runtests.jl:7
   [16] include(::Function, ::Module, ::String) at ./Base.jl:380
   [17] include(::Module, ::String) at ./Base.jl:368
   [18] exec_options(::Base.JLOptions) at ./client.jl:296
   [19] _start() at ./client.jl:506
```

**Comment by krrutkow** [comment]
What version of CBinding are you using?  This was fixed and released in v1.0.7.

**Comment by fleimgruber** [comment]
Thanks for confirming and sorry for the noise, I did not use the correct Julia environment for testing.
